vignettes/cookbook_running_the_analysis.Rmd

```diff
@@ -179,7 +179,7 @@ The `analysis()` function has a number of options that can be set in the `config
 
 All these options are documented in more detail the [section on project parameters](https://rmi-pacta.github.io/pacta.multi.loanbook/articles/config_yml.html#project_parameters) in the `vignette("config_yml")`.
 
-Usually, it will be interesting to run the analysis for more than one by_group, possibly also for multiple combinations of the other parameters. You will therefore have to run the analysis as many times as there are combinations of interest that you wish to generate results for.
+Usually, it will be interesting to run the analysis for more than one `by_group`, possibly also for multiple combinations of the other parameters. You will therefore have to run the analysis as many times as there are combinations of interest that you wish to generate results for.
```

Comment:
note that it could also be more in line with the above text to refer to the conceptual meaning of the parameter `by_group`, rather than explicitly referring to the parameter name?

Reply:
I think it's helpful to have the argument named, to make it more obviously to the user what to change

Reply:
What about hyperlinking the parameter:
"... for more than one [group](https://rmi-pacta.github.io/pacta.multi.loanbook/articles/config_yml.html#by_group)"

Reply:
> What about hyperlinking the parameter documentation: "... for more than one [group](https://rmi-pacta.github.io/pacta.multi.loanbook/articles/config_yml.html#by_group)"

could be 🤷🏻

Reply:
the suggested change is fine for me, hyperlinking as well. Whatever the native speakers consider more appropriate :)

Reply:
how about we name the parameter and link it to the documentation for it

```suggestion
Usually, it will be interesting to run the analysis for more than one [`by_group` value](https://rmi-pacta.github.io/pacta.multi.loanbook/articles/config_yml.html#by_group), possibly also for multiple combinations of the other parameters. You will therefore have to run the analysis as many times as there are combinations of interest that you wish to generate results for.
```